po/he.po

```diff
@@ -464,9 +464,8 @@ msgid ""
 "lists, you can chat to them live on IRC! Join them on #gnucash at irc.gnome."
 "org"
 msgstr ""
-"קל ליצור קשר עם מפתחי גנוקאש. בנוסף למספר לא מועט של רשימות דיוור, אפשר "
-"לשוחח איתם באופן מקוון באמצעות IRC! ניתן להצטרף אליהם בערוץ gnucash# ב־irc."
-"gnome.org"
+"קל ליצור קשר עם מפתחי גנוקאש. בנוסף למספר לא מועט של רשימות דיוור, ניתן לדבר "
+"איתם באופן מקוון דרך IRC! ההצטרפות היא דרך ‎#gnucash ב־irc.gnome.org"
```

Comment:
@yarons I would expect a space, perhaps `ב־ irc.gnome.org`?

Reply:
Hebrew has a single character prefix for the term "in", since it's not joined with a Hebrew word after it, we keep the prefix but add this high hyphen, there are several cases like that, sometimes it breaks links if they are generated automatically but with HTML tags it works flawlessly.

Reply:
Thanks for the explanation!